src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final K[] keys;
+    private final V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_SIZE];
+        values = (V[]) new Object[MAX_SIZE];
+    }
+
+    private boolean objectsEquals(K o1, K o2) {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private boolean isObjectsEquals(Object o1, Object o2) {
```
We usually start with 'is' boolean variables and method names.